dotcom-rendering/src/web/layouts/DecideLayout.tsx

```diff
@@ -33,25 +34,43 @@ export const DecideLayout = ({ CAPI, NAV }: Props): JSX.Element => {
 	// TODO we can probably better express this as data
 	switch (display) {
 		case ArticleDisplay.Immersive: {
-			if (design === ArticleDesign.Interactive) {
-				return (
-					<InteractiveImmersiveLayout
-						CAPI={CAPI}
-						NAV={NAV}
-						format={format}
-						palette={palette}
-					/>
-				);
+			switch (design) {
+				case ArticleDesign.Interactive: {
```

Comment:
Deployment strategy:

1. Deploy DCR rendering all Interactive Immersives as `FullPageInteractiveLayout`
2. Deploy CAPI so that `ArticleDesign.FullPageInteractive` is passed correctly
3. Update DCR to use the `switch (CAPI.isLegacyInteractive)` logic to render either `FullPageInteractiveLayout` or `InteractiveImmersiveLayout`